4.6.2/WcfBasicIISHost/WcfServiceTestClient/Service References/ServiceReference1/Reference.cs

```diff
@@ -1,123 +0,0 @@
-﻿//------------------------------------------------------------------------------
```

Comment:
Now that the service reference is removed, but I did not see the change of program.cs. Does the sample still compile and work?